### DISCUSSION TITLE
Posting job offer

### DISCUSSION THREAD

**Discussion by indominus-hub** [pyvista/pyvista#2458] [Announcements]
Hey guys,

I would like to share a job opening in my company with all of you, but I don't know if it is allowed to do it here.

Administrator/moderator....please let me know

Thanks in advance.

-----

Here is the job description. Hope anyone can find it interesting.

BTW, work permit for Spain is needed. At first, we do not envisage hiring a freelancers, but if no candidates for permanent position are available, that could be also an option...

Best
[software developer job opening English.pdf](https://github.com/pyvista/pyvista/files/8506697/software.developer.job.opening.English.pdf)

**Comment by banesullivan** [comment]
Hi @indominus-hub, thanks for engaging with the PyVista community! Personally, I'm okay with and supportive of you posting a job opening here, so go right ahead and share the details in this thread for all to find!

With that said, this is definitely out of the norm for our discussions and I don't think the maintainer/developer team have thought about this or included anything in the [CODE_OF_CONDUCT.md](https://github.com/pyvista/pyvista/blob/main/CODE_OF_CONDUCT.md) about soliciting services or employment.

I know many of the core developers here have full-time jobs and are quite happy with their current roles (myself included), but it'd be great to share this opportunity with PyVista's community more broadly in case there is a PyVista power-user out there that can help you. I'd also recommend sharing on the PyVista Slack workspace: http://slack.pyvista.org/

Further, I would be *thrilled* if I saw a job posting that had PyVista and VTK listed as desired skills/expertise outside of the niche positions/fields that the core developers (myself included) work in.

**Reply by indominus-hub** [reply]
Hi @banesullivan 

Thanks for your answer.

Here is the job description. Hope anyone can find it interesting.

BTW, work permit for Spain is needed. At first, we do not envisage hiring a freelancers, but if no candidates for permanent position are available, that could be also an option...

Best

[software developer job opening English.pdf](https://github.com/pyvista/pyvista/files/8506693/software.developer.job.opening.English.pdf)

**Reply by banesullivan** [reply]
I updated your original post so others can more easily find it at the top.

This looks like an exciting opportunity and I hope someone here applies!

> Experienced at least with one of the following libraries: vtk.js/trame/vedo/WebGL/three.js/PyVista
 
I love seeing all those Kitware-built or otherwise open-source packages listed!!

**Comment by indominus-hub** [comment]
Thank you very much @banesullivan !

All of them are amazing tools! Hope we can find someone to help us here.

All the best

**Comment by akaszynski** [comment]
Typo:
> postporcessing

**Reply by indominus-hub** [reply]
Thanks! Updated already
